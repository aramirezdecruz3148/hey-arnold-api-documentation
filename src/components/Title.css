@import url('https://fonts.googleapis.com/css?family=Scope+One&display=swap');
@import url('https://fonts.googleapis.com/css?family=Luckiest+Guy&display=swap');

header {
  margin: 0;
  padding: 0;
}

.heroImage {
  background-image: url('../assets/banner-pic.jpg');
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  position: relative;
  height: 45vw;
  width: 100vw;
  margin: 0;
  padding: 0;
}

.heroImage h3 {
  text-align: center;
  position: absolute;
  top: 0;
  left: 50%;
  transform: translate(-50%, -50%);
  font-family: 'Luckiest Guy', cursive;
  color: #FFD90F;
  font-size: 6vw;
  text-shadow:
		-.3vw -.3vw 0 #000,
		.3vw -.3vw 0 #000,
		-.3vw .3vw 0 #000,
		.3vw .3vw 0 #000;
}