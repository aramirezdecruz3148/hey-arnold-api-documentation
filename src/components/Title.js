import React from 'react';

export default function Title() {
  return (
    <header>
      <h3>Hey Arnold API!</h3>
    </header>
  );
}
