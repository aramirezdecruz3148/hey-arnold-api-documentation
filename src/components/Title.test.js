import React from 'react';
import { shallow } from 'enzyme';
import Title from './Title';

describe('Title component', () => {
  it('renders Title', () => {
    const wrapper = shallow(<Title />);
    expect(wrapper).toMatchSnapshot();
  });
});
