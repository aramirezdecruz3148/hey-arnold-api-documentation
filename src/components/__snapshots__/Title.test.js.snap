// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders Title 1`] = `
<header>
  <h3>
    Hey Arnold API!
  </h3>
</header>
`;
